### ISSUE TITLE
Can Pipeline be run without Docker or Singularity?

### ISSUE THREAD

**Issue by AdrianLimaG** [CDCgov/phoenix#45]
Not sure if to put this in bug or enhancement feature, but i don't wish to use docker or singularity to run this analysis, but there does not seem to be any documentation to show how to run this without being virtualized. Can this be run without Docker or Singularity?
 I see an option for Conda in the nextflow config file, but there is not documentation on using this profile.

**Comment by AdrianLimaG** [comment]
Thank you for changing the label, so it is possible to run the pipeline without docker?

**Comment by jvhagey** [comment]
Currently, it is not set up to run one conda environment per process. It just pulls containers (with docker or singularity) for each process. We feel this is best for reproducibility of the pipeline and conda can be unreliable at times. What is the reason for running the pipeline outside of docker or singularity? 

I created just one giant conda environment with mamba with everything in it to run v1.0.0 as follows:

```
mamba create -n phoenix -y \
    bioconda::bbmap==38.96 \
    bioconda::multiqc==1.11 \
    bioconda::fastp==0.23.2 \
    bioconda::mash==2.3 \
    bioconda::mlst==2.22.1 \
    bioconda::fastqc==0.11.9 \
    bioconda::gamma==2.2 \
    bioconda::fastani==1.33 \
    conda-forge::python \
    bioconda::ncbi-amrfinderplus==3.10.40 \
    bioconda::kraken2==2.1.2 \
    bioconda::quast==5.0.2 \
    bioconda::spades==3.15.5 \
    bioconda::krona==2.8.1 \
    bioconda::prokka==1.14.5 \
    conda-forge::biopython \
    conda-forge::rsync \
    conda-forge::bc \
    conda-forge::wget \
    conda-forge::ca-certificates \
    conda-forge::coreutils \
    anaconda::graphviz \
    bioconda::nextflow==22.04.5 \
    conda-forge::gsl==2.7
```

or you can use the [yml file](https://github.com/CDCgov/phoenix/blob/v1.0.1/Dockerfiles/phoenix_v1.0.0.yml) to install the environment like:

`mamba env create -f phoenix_v1.0.0.yml`

You then run PHoeNIx with:

```
conda activate phoenix_v1.0.0
nextflow run cdcgov/phoenix -v1.0.0 -profile test -entry PHOENIX --outdir $PWD/output_folder
```

So basically you don't need to pass it `-profile singularity` or `-profile docker`. This worked on my end to run PHoeNIx let me know if it works for you. I can't guarantee that we will be able to support this method with all future versions of PHoeNIx as it can be hard to get all the programs dependencies to work together so building a giant environment like this kinda just defeats the point of doing things with conda environments/containers.

**Comment by jvhagey** [comment]
Hi @AdrianLimaG can you confirm that this worked for you?

**Comment by AdrianLimaG** [comment]
Hi, @jvhagey yes it was thank you for your help!
